<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDialog</class>
 <widget class="QDialog" name="TestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>关于</string>
  </property>
  <property name="windowIcon">
   <iconset resource="hello.qrc">
    <normaloff>:/logo/Resources/fivision.ico</normaloff>:/logo/Resources/fivision.ico</iconset>
  </property>
  <widget class="QPushButton" name="btn_return">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>570</y>
     <width>121</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>返回主界面</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_zhuashou_left">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>91</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>抓手左移</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_zhuashou_right">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>91</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>抓手右移</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_hand_close">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>91</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>手指夹紧</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_hand_open">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>91</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>手指松开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_boshou_down">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>170</y>
     <width>91</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>拨手退回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_boshou_up">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>91</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>拨手伸出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_boshou_shuiping_down">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>91</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>拨手水平左</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_boshou_shuiping_up">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>110</y>
     <width>91</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>拨手水平右</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_zhuashou_up">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>91</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>抓手上升</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_zhuashou_down">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>91</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>抓手下降</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>520</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>IN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>520</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>OUT</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="hello.qrc"/>
 </resources>
 <connections/>
</ui>
